bokeh/models/annotations.py

```diff
@@ -530,6 +530,52 @@ class BoxAnnotation(Annotation):
 
     """)
 
+class FilledArea(Annotation):
+    ''' Render a filled area band along a dimension.
+
+    '''
+    lower = NumberSpec(help="""
+    The lower coordinates to locate the text anchors.
+    """)
+
+    upper = NumberSpec(help="""
+    The upper coordinates to locate the text anchors.
+    """)
+
+    coords = NumberSpec(help="""
```

Comment:
is this necessary though? I think we just have users provide upper and lower coords as absolute coords, and if they want to tie them to some other base set of coords a transform will work for that.